### ISSUE TITLE
led_vu_config enhancements

### ISSUE THREAD

**Issue by NenuSB** [sle118/squeezelite-esp32#320]
Suggestions for improvement in the VU meters with addressable leds:

    add an LED gain setting in the ESP32 settings in LMS:
    * I never have a player even close to 100% volume, usually~50%, but would like to see more then half of the leds do something.
    * adding extra dummy leds won't work, because the center for L/R would shift
    it doesn't seem to work if no DAC is configured. That's fine, I guess, but would be good to mention in the readme?

I am new to Github; so please tell me if I am doing something wrong with these submissions and/or if I shouldn't even be making them.

**Comment by sle118** [comment]
I suggest @wizmo2 have a look at this

**Comment by wizmo2** [comment]
>  add an LED gain setting in the ESP32 settings in LMS

I'll look into it.

>  it doesn't seem to work if no DAC is configured

What do you use as audio out?

**Comment by NenuSB** [comment]
> What do you use as audio out?

Nothing. Suppose you want to use it purely as a VU meter, synched to some other squeezeboxes. But I don't think that's much of a use case, and I can always add a dummy entry anyway. 
So I don't think that is worth fixing, but maybe a mention in the readme.

**Comment by sle118** [comment]
the default output is i2S and since the data is one way out from the ESP32, there's no need for an actual DAC to be connected.  Now this use case is yet another creative way to use our project!

**Comment by NenuSB** [comment]
> there's no need for an actual DAC to be connected. 

But it is still necessary to define one in the settings, I found.
I entered something like "dac_config: model=I2S,bck=12,ws=32,do=33" without physically connecting anything. 
If you just want to use it for the VU, you should have plenty of spare GPIOs anyway, I guess.

**Comment by sle118** [comment]
@NenuSB yes. To me this is a "hacky" way to use the firmware, since this wasn't one of the use cases we had in mind. I'm glad if it works for you!
